common/scala/src/main/resources/logback.xml

```diff
@@ -13,6 +13,10 @@
   <!-- Kafka -->
   <logger name="org.apache.kafka" level="ERROR" />
 
+  <!-- Zookeeper -->
+  <logger name="org.apache.zookeeper" level="ERROR" />
+  <logger name="org.apache.curator" level="ERROR" />
```

Comment:
May be we do it only for test. Not seen that many zookeeper logs on runtime setups after initial startup is done. Issues related to zookeeper would be rare so better to have logs captured

Reply:
Hm, unsure about the benefits there as errors should be logged either way?

Reply:
The startup logs provide good context at info level around how connection is being established. They prove useful if we face any issue in client connection ... say by co-relating logs by session id from zookeeper side

```
[2018-07-24T13:32:42.583Z] [INFO] Initiating client connection, connectString=exhibitor.marathon.mesos:31886 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@55b5e331
[2018-07-24T13:32:42.600Z] [INFO] Default schema
[2018-07-24T13:32:42.623Z] [INFO] Opening socket connection to server ip-x-x-x-x/x.x.x.x:31886. Will not attempt to authenticate using SASL (unknown error)
[2018-07-24T13:32:42.628Z] [INFO] Socket connection established to ip-x-x-x-x/x.x.x.x:31886, initiating session
[2018-07-24T13:32:42.636Z] [INFO] Session establishment complete on server ip-x-x-x-x/x.x.x.x:31886, sessionid = 0x464cc3ef0eb0004, negotiated timeout = 40000
[2018-07-24T13:32:42.641Z] [INFO] State change: CONNECTED
```